composer.json

```diff
@@ -6,11 +6,13 @@
     "require": {
         "php": "^7.1|^8.0",
         "ext-curl": "*",
+        "guzzlehttp/guzzle": "^7.0",
```

Comment:
I'd recommend `^7.2`. In particular, the 7.0.0 release is actually broken. 7.2.0 is the first version to support PHP 8, too.